addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1080,6 +1080,31 @@ registry._fixAppleCollapse = Animation.extend({
     },
 });
 
+registry.anchorSlide = Animation.extend({
+    selector: 'a[href^="/"][href*="#"]',
+    events: {
+        'click': '_onAnimateClick',
+    },
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     */
+    _onAnimateClick: function (ev) {
```

Comment:
That method is not correct. Image you have a link `/aboutus#top` on your homepage... with your animation you will prevent the user to go to the aboutus page and scroll the homepage to the top...
Maybe the animation should not target link clicks, but URL changes instead.